src/media/ros/ros_file_format.h

```diff
@@ -73,16 +73,42 @@ namespace librealsense
     inline bool convert(const std::string& source, rs2_format& target)
     {
         bool ret = true;
-        if (source == sensor_msgs::image_encodings::MONO16)     target = RS2_FORMAT_Z16;
+        std::string source_alias("");
+        bool source_alias_assignement = false;
+        if (source == sensor_msgs::image_encodings::MONO16) {
+            target = RS2_FORMAT_Z16;
+            source_alias_assignement = true;
+        }
+        if (source == sensor_msgs::image_encodings::TYPE_8UC1) {
```

Comment:
It is not clear why there is a need to handle RGB8 and Y16 separately - let's discuss it

Reply:
discussed successfully